api/v1beta1.go

```diff
@@ -19,16 +19,21 @@ func setupV1Beta1Router(router *mux.Router, handlers *Handlers) *mux.Router {
 		Methods(http.MethodGet).
 		HandlerFunc(handlers.Search.Suggest)
 
-	router.PathPrefix("/lineage/{id}").
+	router.PathPrefix("/lineage/{urn}").
 		Methods(http.MethodGet).
-		HandlerFunc(handlers.Lineage.GetLineage)
+		HandlerFunc(handlers.LineageV2.GetGraph)
```

Comment:
@StewartJingga if we are going to remove a whole functionality of old Lineage, wouldn't it be better to remove the package as well? So we don't have `lineage/v1` and `lineage/v2`. Package `lineage` will only have the latest version one.

Reply:
+1

Reply:
yeah missed that, thanks for catching that, removed!